File: javascript/write.js
export function draw() {
    let com1width = Math.floor(num / 10);
    for (let i = 0; i < num; i++) {
        console.log("L".repeat(com1width));
    }
    let com2width = Math.floor(num / 5);
    let com2height = Math.floor(num / 50);
    for (let i = 0; i < com2height; i++) {
        console.log("L".repeat(com2width))
    }
}

File: python/write.py
import math

def draw() -> None:
    num: int = 2147483647

    com1width: int = math.floor(num / 20)
    for _ in range(0, num):
        print(com1width * "L")
    com2width: int = math.floor(num / 5)
    com2height: int = math.floor(num / 50)
    for _ in range(0, com2height):
        print(com2width * "L")
